clients/src/main/java/org/apache/kafka/clients/admin/ExpireDelegationTokenOptions.java

```diff
@@ -28,6 +28,13 @@
 public class ExpireDelegationTokenOptions extends AbstractOptions<ExpireDelegationTokenOptions> {
     private long expiryTimePeriodMs = -1L;
 
+    /**
+     * @param expiryTimePeriodMs the time period until we should expire this token.
+     * {@code expiryTimestamp} is the time we actually expire the token.
+     * If {@code expiryTimePeriodMs} < 0, token will be expired immediately.
+     * If {@code expiryTimePeriodMs} >= 0 and {@code expiryTimestamp} is not due,
+     * {@code expiryTimestamp} will be set to {@code min(expiryTimestamp, maxTimestamp)}.
```

Comment:
I feel `min(expiryTimestamp, maxTimestamp)` is a bit puzzling, since the public APIs don't mention the `maxTimestamp`. How about:

```
{@code expiryTimePeriodMs} >= 0: the token will update the `expiration timestamp` if the current expiration timestamp is small than (now + expiryTimePeriodMs)
```

Reply:
okay, I'll change it to 

```
    /**
     * @param expiryTimePeriodMs the time period until we should expire this token.
     * {@code expiryTimePeriodMs} >= 0: the token will update the `expiration timestamp` if the current expiration timestamp is small than (now + expiryTimePeriodMs).
     * {@code expiryTimePeriodMs} < 0: token will be expired immediately.
     */
```